FaunaDB.Client/Types/BytesV.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+using FaunaDB.Query;
+using Newtonsoft.Json;
+
+namespace FaunaDB.Types
+{
+    /// <summary>
+    /// A FaunaDB bytes type.
+    /// <para>
+    /// See <see href="https://fauna.com/documentation/queries#values-special_types">FaunaDB Special Types</see>.
+    /// </para>
+    /// </summary>
+    public class BytesV : Value
+    {
+        byte[] _Value;
+
+        public byte[] Value
+        {
+            get
+            {
+                return (byte[])_Value.Clone();
+            }
+        }
+
+        internal BytesV(string base64)
+        {
+            _Value = FromUrlSafeBase64(base64);
+        }
+
+        public BytesV(params byte[] value)
```

Comment:
Does this constructor allow for an array of bytes to come in instead of a varargs?